docs/changelog.md

```diff
@@ -30,6 +30,10 @@ permalink: /changelog/
      intended for public use, part of the glean API has been renamed from
      `mozilla.components.service.glean.metrics` to
      `mozilla.components.service.glean.private`.
+   * ⚠️ **This is a breaking API change**: Labeled metrics are now their own
```

Comment:
I guess this (as with some other changes) come from the fact that you're based off some other local branch?

Reply:
Yeah -- I still need to get the labeled stuff merged first (hence the WIP on this).  That should clear out before I finally merge.